encass-annotations/src/main/gateway/config/encass.yml

```diff
@@ -0,0 +1,32 @@
+encass_with_annotation:
+  annotations:
+  - type: "@bundle"
```

Comment:
Use annotation in a simple way i.e., 
**annotations:
  - "@bundle"**

Reply:
If i dont give name its takes the file name as project-name- entity name , but build command expects different file name
As result build task is failing, so configured name

Reply:
is it an issue? how was it working for "hello-world" encas?